OutOfSchool/OutOfSchool.WebApi/Models/Child/ChildBaseDTO.cs

```diff
@@ -1,16 +1,11 @@
-using System;
-using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
-using System.Text.Json.Serialization;
 using OutOfSchool.Services.Enums;
 using OutOfSchool.WebApi.Models.SocialGroup;
 
 namespace OutOfSchool.WebApi.Models;
 
-public class ChildDto
+public class ChildBaseDto
```

Comment:
If i remember correctly, to create a child we don't need `List<SocialGroupDto>` but `List<long>` of social group Ids. Maybe move it out of base and have these lists in their own normal/create/update entities?

Reply:
done